Art/shag1802/style.css

```diff
@@ -0,0 +1,68 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 100vh;
+    background-color: #0c002b;
+}
+
+.button{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 400px;
+    height: 100px;
+    /* background-color: red; */
+    /* padding: 25px 45px; */
+    position: relative;
+    overflow: hidden;
+    border: 1px solid #1779ff;
+    border-radius: 20px;
+}
+
+.button a{
+    text-decoration: none;
+    color: white;
+    font-size: 48px;
+    transition: 0.5s letter-spacing , 0.5s color;
+    /* overflow: hidden; */
+}
+.button::after{
+    content :"";
+    width: 100%;
+    height: 50%;
+    background-color: #1779ff;
+    position: absolute;
+    top: 0;
+    left: 0;
+    transform: translateX(100%) translateY(-100%);
+    transition: 0.5s transform;
+    z-index: -1;
+}
+.button::before{
+    content :"";
+    width: 100%;
+    height: 50%;
+    background-color: #1779ff;
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    transform: translateX(-100%) translateY(+100%);
+    transition: 0.5s transform;
+    z-index: -1;
+}
+.button:hover::after,.button:hover::before{
+    transform: translateX(0%)  translateY(0%);
+}
+.button:hover a{
+    letter-spacing: 2px;
+    color: #0c002b;
+}
+/* .button:hover ::after{
+    transform: translateX(-100%) translateY(100%);
+} */
```

Comment:
# Feedback - Unnecessary comment
- [ ]  please remove the comment

Reply:
Please review again , I have removed the unnecessary comments from the code.  Thank you